automation/taskcluster/lib/tasks.py

```diff
@@ -22,7 +22,7 @@ def build_task(self, name, description, command, artifacts={}, scopes=[], featur
         return self._craft_default_task_definition(
             worker_type=self.build_worker_type,
             provisioner_id='aws-provisioner-v1',
-            dependencies=[self.task_id],
```

Comment:
This dep is actually added here https://github.com/mozilla-mobile/reference-browser/pull/645/files#diff-59f90d509bc6ae2c9d877250c5ba49c5L127 (I had forgotten about that)